cookbook/recipes/nodejs.rb

```diff
@@ -19,14 +19,14 @@
 #
 
 apt_repository 'node_0.12' do
-  uri 'https://deb.nodesource.com/node_0.12'
+  uri 'https://deb.nodesource.com/node_10.x/'
```

Comment:
Upgraded the node version due to many warnings and errors when guardian tried to `npm install`. Figured this may solve those issues, and it did